spec/flipper/adapters/sequel_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'helper'
+require 'sequel'
+
+Sequel::Model.db =  Sequel.sqlite
+Sequel.extension :migration, :core_extensions
+
+require 'flipper/adapters/sequel'
+require 'generators/flipper/templates/sequel_migration'
+require 'flipper/spec/shared_adapter_specs'
+
+RSpec.describe Flipper::Adapters::Sequel do
+  subject do
+    described_class.new(feature_class: feature_class,
+                        gate_class: gate_class)
+  end
+
+	let(:feature_class) { Flipper::Adapters::Sequel::Feature }
```

Comment:
Formatting looks a little odd here. Would you mind moving this line and the next one back a few characters to line it up with the subject and before blocks?